### ISSUE TITLE
Component compatibility with lightning web security

### ISSUE THREAD

**Issue by gburnett** [salesforce/design-system-react#3044]
This is possibly related to the platform rather than this component library but I think that it is worth asking here.

Does anybody know if the design system react components are tested against, or compatible with, [lightning web security](https://developer.salesforce.com/docs/component-library/documentation/en/lwc/lwc.security_lwsec_intro)?

I have asked a [question](https://salesforce.stackexchange.com/questions/376923/how-do-i-use-design-system-react-modal-with-lightning-web-security) on the salesforce stack exchange to explain in more detail.

The short version is that I have a react app that uses the modal from this library.  I bundle that app as an LWC static resource.  Attempting to open the modal, in an org with LWS enabled, will cause the LWC to fail.

**Comment by interactivellama** [comment]
The React render library is not allowed in Lightning--outside of a workaround such as an `iframe`.

**Comment by gburnett** [comment]
> The React render library is not allowed in Lightning ...

@interactivellama  When you say `Lightning` here are you referring to LWS or LWC?
